draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1119,21 +1114,115 @@ SDP Answer
           from SSRC to identification-tag is received. Note that the mapping
           from SSRC to identification-tag can change at any time during an RTP
           session.
+	</t>
+
+
+      <t>Note: The following algorithm does not yet have WG consensus
+      but is included here as something concrete for the working group
+      to discuss.</t>
+
+      <t>To prepare for demultiplexing RTP packets to the correct "m="
+      line, the following steps MUST be followed for each BUNDLE
+      group.</t>
+
+      <t>
+        <list>
+
+          <t>Construct a table mapping MID to "m=" line for each
+          "m=" line in this BUNDLE group.</t>

```

Comment:
Add comment: "Each MID maps to only one m= line, and each m= line maps to only one MID".